<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>499</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>671</width>
      <height>381</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <property name="spacing">
      <number>6</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="font">
          <font>
           <pointsize>18</pointsize>
          </font>
         </property>
         <property name="title">
          <string>X</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="1" column="0">
             <widget class="QPushButton" name="buzz_down_x">
              <property name="minimumSize">
               <size>
                <width>60</width>
                <height>60</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>80</width>
                <height>80</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>Wingdings 3</family>
                <pointsize>18</pointsize>
               </font>
              </property>
              <property name="text">
               <string>H</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QPushButton" name="step_down_x">
              <property name="minimumSize">
               <size>
                <width>60</width>
                <height>60</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>80</width>
                <height>80</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>Wingdings 3</family>
                <pointsize>18</pointsize>
               </font>
              </property>
              <property name="text">
               <string>f</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QPushButton" name="step_up_x">
              <property name="minimumSize">
               <size>
                <width>60</width>
                <height>60</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>80</width>
                <height>80</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>Wingdings 3</family>
                <pointsize>18</pointsize>
               </font>
              </property>
              <property name="text">
               <string>g</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QPushButton" name="buzz_up_x">
              <property name="minimumSize">
               <size>
                <width>60</width>
                <height>60</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>80</width>
                <height>80</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>Wingdings 3</family>
                <pointsize>18</pointsize>
               </font>
              </property>
              <property name="text">
               <string>I</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QFormLayout" name="formLayout">
            <item row="0" column="0">
             <widget class="QLabel" name="voltageLabel">
              <property name="font">
               <font>
                <pointsize>8</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Voltage</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QDoubleSpinBox" name="voltage_x">
              <property name="font">
               <font>
                <pointsize>8</pointsize>
               </font>
              </property>
              <property name="minimum">
               <double>15.000000000000000</double>
              </property>
              <property name="maximum">
               <double>50.000000000000000</double>
              </property>
              <property name="value">
               <double>35.000000000000000</double>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="frequencyLabel">
              <property name="font">
               <font>
                <pointsize>8</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Frequency</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QSpinBox" name="frequency_x">
              <property name="font">
               <font>
                <pointsize>8</pointsize>
               </font>
              </property>
              <property name="accelerated">
               <bool>true</bool>
              </property>
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>10000</number>
              </property>
              <property name="value">
               <number>300</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QComboBox" name="speed_x">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <item>
             <property name="text">
              <string>Normal</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Slow</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Fast</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>User set</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="font">
          <font>
           <pointsize>18</pointsize>
          </font>
         </property>
         <property name="title">
          <string>Y</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <layout class="QGridLayout" name="gridLayout_4">
            <item row="1" column="0">
             <widget class="QPushButton" name="step_down_y">
              <property name="minimumSize">
               <size>
                <width>60</width>
                <height>60</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>80</width>
                <height>80</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>Wingdings 3</family>
                <pointsize>18</pointsize>
               </font>
              </property>
              <property name="text">
               <string>i</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QPushButton" name="step_up_y">
              <property name="minimumSize">
               <size>
                <width>60</width>
                <height>60</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>80</width>
                <height>80</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>Wingdings 3</family>
                <pointsize>18</pointsize>
               </font>
              </property>
              <property name="text">
               <string>h</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QPushButton" name="buzz_up_y">
              <property name="minimumSize">
               <size>
                <width>60</width>
                <height>60</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>80</width>
                <height>80</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>Wingdings 3</family>
                <pointsize>18</pointsize>
               </font>
              </property>
              <property name="text">
               <string>J</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QPushButton" name="buzz_down_y">
              <property name="minimumSize">
               <size>
                <width>60</width>
                <height>60</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>80</width>
                <height>80</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>Wingdings 3</family>
                <pointsize>18</pointsize>
               </font>
              </property>
              <property name="text">
               <string>K</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QFormLayout" name="formLayout_3">
            <item row="0" column="0">
             <widget class="QLabel" name="voltage_label_y">
              <property name="font">
               <font>
                <pointsize>8</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Voltage</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QDoubleSpinBox" name="voltage_y">
              <property name="font">
               <font>
                <pointsize>8</pointsize>
               </font>
              </property>
              <property name="minimum">
               <double>15.000000000000000</double>
              </property>
              <property name="maximum">
               <double>50.000000000000000</double>
              </property>
              <property name="value">
               <double>35.000000000000000</double>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="frequencyLabel_3">
              <property name="font">
               <font>
                <pointsize>8</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Frequency</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QSpinBox" name="frequency_y">
              <property name="font">
               <font>
                <pointsize>8</pointsize>
               </font>
              </property>
              <property name="accelerated">
               <bool>true</bool>
              </property>
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>10000</number>
              </property>
              <property name="value">
               <number>300</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QComboBox" name="speed_y">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <item>
             <property name="text">
              <string>Normal</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Slow</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Fast</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>User set</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="font">
          <font>
           <pointsize>18</pointsize>
          </font>
         </property>
         <property name="title">
          <string>Z</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QGridLayout" name="gridLayout_3">
            <item row="1" column="0">
             <widget class="QPushButton" name="step_down_z">
              <property name="minimumSize">
               <size>
                <width>60</width>
                <height>60</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>80</width>
                <height>80</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>Wingdings 3</family>
                <pointsize>18</pointsize>
               </font>
              </property>
              <property name="text">
               <string>i</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QPushButton" name="step_up_z">
              <property name="minimumSize">
               <size>
                <width>60</width>
                <height>60</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>80</width>
                <height>80</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>Wingdings 3</family>
                <pointsize>18</pointsize>
               </font>
              </property>
              <property name="text">
               <string>h</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QPushButton" name="buzz_up_z">
              <property name="minimumSize">
               <size>
                <width>60</width>
                <height>60</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>80</width>
                <height>80</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>Wingdings 3</family>
                <pointsize>18</pointsize>
               </font>
              </property>
              <property name="text">
               <string>J</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QPushButton" name="buzz_down_z">
              <property name="minimumSize">
               <size>
                <width>60</width>
                <height>60</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>80</width>
                <height>80</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>Wingdings 3</family>
                <pointsize>18</pointsize>
               </font>
              </property>
              <property name="text">
               <string>K</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QFormLayout" name="formLayout_2">
            <item row="0" column="0">
             <widget class="QLabel" name="voltageLabel_2">
              <property name="font">
               <font>
                <pointsize>8</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Voltage</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QDoubleSpinBox" name="voltage_z">
              <property name="font">
               <font>
                <pointsize>8</pointsize>
               </font>
              </property>
              <property name="minimum">
               <double>15.000000000000000</double>
              </property>
              <property name="maximum">
               <double>50.000000000000000</double>
              </property>
              <property name="value">
               <double>35.000000000000000</double>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="frequencyLabel_2">
              <property name="font">
               <font>
                <pointsize>8</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Frequency</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QSpinBox" name="frequency_z">
              <property name="font">
               <font>
                <pointsize>8</pointsize>
               </font>
              </property>
              <property name="accelerated">
               <bool>true</bool>
              </property>
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>10000</number>
              </property>
              <property name="value">
               <number>300</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QComboBox" name="speed_z">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <item>
             <property name="text">
              <string>Normal</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Slow</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Fast</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>User set</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="stop_all">
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Stop All</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>720</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <action name="actionClose">
   <property name="icon">
    <iconset>
     <normaloff>../../../artwork/icons/oxygen/22x22/application-exit.png</normaloff>../../../artwork/icons/oxygen/22x22/application-exit.png</iconset>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
   <property name="toolTip">
    <string>Close</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
